### DISCUSSION TITLE
Communication between PC and device

### DISCUSSION THREAD

**Discussion by Deicidezz** [bitcraze/discussions#1373] [Q&A]
Hello, I'm using Crazyflie2.1&AI-Deck1.1 as developing platform, I want to send custom data content(for example , the content is '123456qwer') to Crazyflie through Crazyradio on Windows, how can realize it. I have read the [API reference for CFLib] but still fail.

**Comment by ataffanel** [comment]
I will need a bit more information and context about what you are doing in order to be able to help.

For instance:
 - Are you developping a program in python using the lib already?
 - Are you writing an APP in the Crazyflie?

Basically what is your setup and what are you trying to achieve, Intuitively, the app channel is what you should look at, but it really depends of the context.
